libflux/flux-core/src/semantic/types.rs

```diff
@@ -348,6 +364,7 @@ impl From<Record> for MonoType {
 }
 
 impl MonoType {
+    #[allow(missing_docs)]
```

Comment:
Unify performs unification on the type with another type. If successful it results in a solution to the unification problem, in the form of a substitution. If there is no solution to the unification problem then unification fails and an error is reported.